### PR TITLE
Reducer and action

### PR DESCRIPTION
**Pull request by EzekielUtshudi**
- [ ] Add React Redux (npm install react-redux).
- [ ] application files using a "feature folder" approach 
- [ ] actions and reducer (in /src/redux/books/books.js
- [ ] categories actions and reducer
- [ ] Configure the Redux Store (/src/redux/configureStore.js):

### CODE REVIEW

File: src/redux/Categories/categories.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// Redux
```
Comment (ShadyShawkat) [inline src/redux/Categories/categories.js:1]: - [x] Kindly make sure to implement the categories actions and reducers, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Write%20your%20categories,the%20current%20state.) requirement.

File: src/redux/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// configureStore.js
+
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './Books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+  // additional reducers could be added here
```
Comment (ShadyShawkat) [inline src/redux/configureStore.js]: - [ ] Please make sure to include the categories reducer here after you implement it in its respective file.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @EzekielUtshudi, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @EzekielUtshudi👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**